### PR TITLE
GPT section

### LINKED ISSUE #14: Write GPT2 section

### PR DESCRIPTION
**Pull request by EhrenbruderRoschtbraten**
Wrote the GPT section.

closes #14

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -64,6 +64,15 @@ \subsection{Generating models}
 \subsubsection{Image generating models}
 
 \subsubsection{Text generating models}
+We choose the GPT model as one possibility to generate texts for testing our augmentation methods. The GPT model tries to counter the scarcity of labeled text data for machine learning. 
+
+Radford et. al try to avoid this problem in a simple two step procedure:
```
Comment (RaphaW) [inline paper.tex]:
```suggestion
Radford et. al try to avoid this problem in a simple two step procedure:
```

I would do a citation here as well

**Review by RaphaW — APPROVED** [review]
lgtm
I have one recommondation

**Review by 1Maxnet1 — COMMENTED** [review]
I would add the Version (aka 2) to the section to clarify that we used the GPT2 model

**Review by 1Maxnet1 — APPROVED** [review]
LGTM

**Pull request merged**